### ISSUE TITLE
Space above tabs does not react to clicks

### ISSUE THREAD

**Issue by xnoreq** [microsoft/terminal#8363]
# Environment

```none
Microsoft Windows [Version 10.0.19042.630]
Windows Terminal Preview
Version: 1.5.3242.0
```

# Steps to reproduce

Click above tabs.

# Expected behavior

- In maximized state, since tabs visually extend to the top of the title bar, clicking at the very top should also activate the tab.
- In normal state, either make tabs expand visually and behave the same as in maximized state, or make the title bar above the tabs clickable.

# Actual behavior

- In maximized state, tabs visually extend to the top of the title bar, but clicking at the top does not activate the tab.
- In normal state, tabs visually don't extend to the top of the title bar but leave a small area of the title bar. Clicking this area doesn't do anything, the window cannot be dragged. Double-click doesn't maximize the window.

**Comment by Don-Vito** [comment]
@xnoreq - might be related to https://github.com/microsoft/terminal/issues/7422.

**Comment by zadjii-msft** [comment]
> In maximized state, tabs visually extend to the top of the title bar, but clicking at the top does not activate the tab.

I'm definitely not seeing this - any chance you're using an auto-hiding taskbar?

~I believe the other half of this bug is in #1625~ Nope, okay it's not in there. I know there's one on the backlog somewhere for the area above the tabs, but idk where.

**Comment by cactysman** [comment]
> I'm definitely not seeing this - any chance you're using an auto-hiding taskbar? What display resolution are you at? Monitor layout?

I got the same issue every now and then and was about to open an issue myself until I saw this.

I was able to reproduce it on both 1080p and 4k resolutions (100% and 150% scaling) (didn't test others so far), with a pretty standard monitor layout with a screen left and right with no offsets, but it only happens sometimes (I don't really know what could be the reason).

I will try to find a definite setup that guarantees that this is happening.
